### ISSUE TITLE
Name property of a focusable element must not be null (Scrollable Input)

### ISSUE THREAD

**Issue by MicrosoftIssueBot** [microsoft/calculator#482]
**Repro Steps**
* Click on the Menu options
* Click on Programmer options
* Chose “BIN” from the list
* Enter number 1 (around >50 times) in the input field to invoke the scroll bar
* Observe Hyperlink for scrolling right/left is missing an accessible name
* Observe the issue using inspect.exe (available in C:\Program Files (x86)\Windows Kits\10\bin\<SDK Version>\x64)

**Actual Result**
Name Property is not defined for left/right buttons.

**Expected Result**
Name Property should be defined

![image](https://user-images.githubusercontent.com/1414423/56937408-44f25a80-6ab1-11e9-8f9b-4c303617605a.png)

**Comment by MicrosoftIssueBot** [comment]
This is your friendly Microsoft Issue Bot. I created this issue automatically as requested by a team member.

**Comment by MelulekiDube** [comment]
Hello,

I am a newbie and Interested in working on this issue.

**Comment by grochocki** [comment]
Thanks, @MelulekiDube, it is all yours! Let us know if you have any questions. For more information on making changes to the code, check out our [contributing guidelines](https://github.com/microsoft/calculator/blob/master/CONTRIBUTING.md#git-workflow).

**Comment by MelulekiDube** [comment]
@grochocki Is a visual test using the inspection tool on the buttons enough to pass as a test for this issue?

**Comment by danbelcher-MSFT** [comment]
Hey @MelulekiDube, using Inspect should be sufficient.  One of the repo maintainers will build your branch and verify the fix before merging.
